services/api/src/main.py

```diff
@@ -37,36 +37,59 @@
 
 app = Flask(__name__)
 
+
+ENABLE_RSU_FEATURES = os.environ.get("ENABLE_RSU_FEATURES", "true") != "false"
+ENABLE_INTERSECTION_FEATURES = (
+    os.environ.get("ENABLE_INTERSECTION_FEATURES", "true") != "false"
+)
+ENABLE_WZDX_FEATURES = os.environ.get("ENABLE_WZDX_FEATURES", "true") != "false"
+
 smtp_error_handler.configure_error_emails(app)
 
 app.wsgi_app = Middleware(app.wsgi_app)
 api = Api(app)
 
 api.add_resource(HealthCheck, "/")
 api.add_resource(UserAuth, "/user-auth")
-api.add_resource(RsuInfo, "/rsuinfo")
-api.add_resource(RsuOnlineStatus, "/rsu-online-status")
-api.add_resource(RsuQueryCounts, "/rsucounts")
-api.add_resource(RsuQueryMsgFwd, "/rsu-msgfwd-query")
-api.add_resource(RsuCommandRequest, "/rsu-command")
-api.add_resource(RsuMapInfo, "/rsu-map-info")
-api.add_resource(RsuGeoQuery, "/rsu-geo-query")
-api.add_resource(WzdxFeed, "/wzdx-feed")
-api.add_resource(RsuGeoData, "/rsu-geo-msg-data")
-api.add_resource(IssScmsStatus, "/iss-scms-status")
-api.add_resource(RsuSsmSrmData, "/rsu-ssm-srm-data")
-api.add_resource(AdminNewRsu, "/admin-new-rsu")
-api.add_resource(AdminRsu, "/admin-rsu")
-api.add_resource(AdminNewIntersection, "/admin-new-intersection")
-api.add_resource(AdminIntersection, "/admin-intersection")
+if ENABLE_RSU_FEATURES:
```

Comment:
Is it maybe worth including all of the relevant features for each unique feature flag under a single 'if' statement instead of breaking them down into a dozen?

Reply:
updated!